xwiki-platform-core/xwiki-platform-web/xwiki-platform-web-war/src/main/webapp/resources/js/xwiki/viewers/attachments.js

```diff
@@ -256,6 +256,16 @@ require(['jquery', 'xwiki-events-bridge'], function($) {
   $(document).on('xwiki:html5upload:done', function() {
     updateCount();
   });
+
+  /**
+   * Call "updateCount" when the attachment tab content gets loaded, in order to make sure that the counter
+   * reflects the actual number of attachments, which may have changed since the initial page load.
+   */
+  document.observe("xwiki:docextra:loaded", function(event) {
```

Comment:
Why use Prototype.js in a Require block with jQuery dependency? See a few lines above. See also https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/FrontendResources/JavaScriptAPI/#HBridgingcustomXWikieventsbetweenPrototypeandjQuery .

Reply:
Indeed, a lack of observation by me, thank you for the review, the latest commit uses jQuery instead.